formulaire-public/src/main/resources/public/ts/controllers/captcha.ts

```diff
@@ -65,6 +66,7 @@ export const captchaController = ng.controller('CaptchaController', ['$scope',
     vm.doSend = async () : Promise<void> => {
         let distributionData = await responseService.sendResponses(vm.formKey, vm.distributionKey, vm.responseCaptcha, vm.responses);
         let distribution = Mix.castAs(Question, distributionData);
+        vm.isProcessing = true;
```

Comment:
Faut le mettre au début de la fonction parce que si tu spam clic il va call plusieurs fois le responseService.sendResponses()

```suggestion
        vm.isProcessing = true;
        let distributionData = await responseService.sendResponses(vm.formKey, vm.distributionKey, vm.responseCaptcha, vm.responses);
        let distribution = Mix.castAs(Question, distributionData);
```